packages/opentelemetry-cloud-trace-exporter/test/exporter.test.ts

```diff
@@ -44,6 +44,21 @@ describe('Google Cloud Trace Exporter', () => {
     });
   });
 
+  describe('_getClient', () => {
```

Comment:
Usually private methods are not unit tested, because they are implementation details, and any problems with them should be caught in tests of public methods. However, since `_getClient` is mocked in the 'export' describe block, the logic is not being tested at all, so I see why we might want this test. My suggestion would be instead of mocking `_getClient`, mock `google.devtools.cloudtrace.v2.TraceService` so that `new TraceService` will return the mock object that is currently returned from the mock `_getClient`. Then more of the real code will run when the 'should export spans' it block is run. You could also verify in that test that `new TraceService` is called with the correct two arguments (URL and credentials). Then I would remove this '_getClient' describe block and test. But let me know if you have any other thoughts/opinions on that.

Reply:
Yea this is a good suggestion. Will add